www/components/MeetingNameDate.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable indent */
+/* eslint-disable no-trailing-spaces */
+// David
+// This component has a flag icon which is white while meeting
+// is being edited and red when saved,
```

Comment:
The flag icon indicates the priority of the meeting, maybe we should highlight it differently if it is being edited or just add "editing" in front of the title.

Reply:
ok i just mistook why it was white in one figma and red in another.  i dont think we need to say editing it will be obvious